### ISSUE TITLE
Problem with autolevel menu

### ISSUE THREAD

**Issue by marcelobsilva18** [makerbase-mks/MKS-EAGLE#2]
When selecting autolevel, the machine does the autolevel using the bltouch but the menu doesn't jump into auto-leveling and doesn't jump intoBL-touch probe menu to set Zoffset. I followed your video on youtube and checked the details here on your github but I can't find the answer. Please help!

**Comment by mks-viva** [comment]
Like this: https://github.com/makerbase-mks/MKS-Robin-Nano-V3.X/issues/73

**Comment by marcelobsilva18** [comment]
Yes I checked that (I have also commented on it) but I still don't get the menu for the z-offset after the autobedleveling is done. That solution just allows to override the settings on the machine, it doesn't work the same way. Is there any option missing from the configuration?

**Comment by marcelobsilva18** [comment]
> Like this: [makerbase-mks/MKS-Robin-Nano-V3.X#73](https://github.com/makerbase-mks/MKS-Robin-Nano-V3.X/issues/73)

Can you please tell me something about this? Do you need my configurations?

**Comment by mks-viva** [comment]
Your source code provide to me.

**Comment by marcelobsilva18** [comment]
Here is my code:
[Marlin.zip](https://github.com/makerbase-mks/MKS-EAGLE/files/7668180/Marlin.zip)

**Comment by Jcm0724** [comment]
I've almost the same issue even define AUTO_BED_LEVELING_BILINEAR. No autolevel setting... I've done everything you've done but still nothing..no autolevel menu showing up.

**Comment by just-trey** [comment]
I have the same issue. It does not switch to the z offset menu. I do have the auto-level menu and it goes through and probes the bed, but in my previous marlin 1.x setup it would probe the bed, center the nozzle, bring it down to the z offset, then the z offset menu would appear. Also in the previous version, I had a z offset menu setting under more.. but I do not see it,  Is there a way to get to this screen?

**Comment by Smarthelpers** [comment]
Did anyone found a solution? Why MKS don't help us with this problem?

**Comment by Judsonian** [comment]
Bump ....

**Comment by Judsonian** [comment]
For now you must use the firmware version 2.0 that is available through MKS (not the base Marlin). They've done some magic in the code to make it work.
